src/main/webapp/login-script.js

```diff
@@ -8,8 +8,18 @@ async function signIn(googleUser) {
   const id = await response.json();
   if (id.verified) {
     // Successful sign-in.
-    document.getElementById("class-info").classList.remove("hidden");
-    document.getElementById("login-box").classList.add("hidden");
+    document
```

Comment:
Since you're doing pretty much the same thing here in multiple places, how about pulling this out into a function of its own?